web/templates/web/children-list.html

```diff
@@ -13,6 +12,43 @@
 {% endblock %}
 {% block content %}
     <script>
+        $(document).ready(function() {
+            $('.datepicker').datepicker({})
+            .on('change', function () {
+               var age = getAge(this);
+               if (document.getElementById('age_calc')) {
+                   document.getElementById('age_calc').innerHTML = `Age: ${age}`;
+               } else {
+                   var x = document.createElement("p");
+                   x.setAttribute("id", "age_calc");
+                   x.setAttribute("class", "age_format")
+                   var t = document.createTextNode(`Age: ${age}`);
+                   x.appendChild(t);
+                   document.getElementsByClassName("form-group")[1].insertBefore(x,document.getElementById("id_birthday"));
+               }
+           });
+        });
+        function getAge(dateValue) {
```

Comment:
@pattisdr I would use moment.js but keep the same functionality you have now. Just use it for the calculations so you don't have to worry about leap years or any of that nonsense.